apps/sms/services/js/bridge_service_mixin.js

```diff
@@ -35,9 +37,20 @@
         service.method(exposedMethod, this[exposedMethod].bind(this));
       });
 
-      this[priv.streams].forEach((exposedStream) => {
-        service.stream(exposedStream, this[exposedStream].bind(this));
-      });
+      var streams = this[priv.streams];
+      if (streams.length > 0) {
```

Comment:
I was thinking about moving this piece of code to a separate function ```initStreams``` or something, but was a bit lazy and not sure if it's really needed :) What do you think?